clients/src/test/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManagerTest.java

```diff
@@ -1034,7 +1209,8 @@ protected void checkDisconnects(final long currentTimeMs) {
                         FutureCompletionHandler handler = unsentRequest.handler();
                         AuthenticationException authenticationException = client.authenticationException(node);
                         long startMs = unsentRequest.timer().currentTimeMs() - unsentRequest.timer().elapsedMs();
-                        handler.onComplete(new ClientResponse(makeHeader(unsentRequest.requestBuilder().latestAllowedVersion()),
+                        handler.onComplete(new ClientResponse(
+                                makeHeader(unsentRequest.requestBuilder().latestAllowedVersion()),
```

Comment:
nit: This would be neater with the `new ClientResponse(` on the new line in its entirety.